kube/src/lib.rs

```diff
@@ -62,9 +62,12 @@
 //! #[derive(CustomResource, Deserialize, Serialize, Clone, Debug, Validate, JsonSchema)]
 //! #[kube(group = "clux.dev", version = "v1", kind = "Foo", namespaced)]
 //! pub struct FooSpec {
+//!     #[garde(skip)]
 //!     info: String,
 //!     #[validate(length(min = 3))]
+//!     #[garde(length(min = 3))]
```

Comment:
Slightly sad about this as these attributes are identical, but it does technically provide us with a new way to separate client-side and server-side validation (if that is useful), even though we were primarily using it as a way to hook into schemars.

Not sure if there's anything schemars or garde can do about this. I saw no mention of [schemars in garde](https://github.com/jprochazk/garde/search?q=schemars), and similarly no mention of the [garde in schemars](https://github.com/GREsau/schemars/search?q=garde).

There is [this doc](https://github.com/GREsau/schemars/blob/73715c10e0007c766969732a1ff53f358c807780/docs/1.1-attributes.md#supported-validator-attributes) that shows that we can use the `#[schemars(length(min = 3))]` instead to highlight that this is a schema-only feature which might be worth doing to avoid confusion over the fork.

Reply:
Yeah, it is a bit unfortunate. It's nice that we get a separation, but it was also really cool to have just one attribute that can be used for validation on both sides. Maybe at some point something can be PR'd upstream to support `#[garde]` in schemars.

Great find with the doc! I worked completely off the git history and was under the impression they only support `#[validate]`. Will make the changes 👍🏻